app/main.py

```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name, age):
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    pupil_array = [Person(person["name"], person["age"]) for person in people]
+
+    for persons in people:
```

Comment:
Could you rename `persons` -> `person`. You store a single person in it.

Reply:
okay)